desktop/cmp/chart/ChartModel.js

```diff
@@ -14,28 +14,23 @@ import {observable, action} from '@xh/hoist/mobx';
 @HoistModel
 export class ChartModel {
 
-    @observable.ref config = {};
-    @observable.ref series = [];
+    @bindable.ref highchartsConfig = {};
+    @bindable.ref series = [];
 
     /**
      * @param {Object} c - ChartModel configuration.
-     * @param {Object} c.config - Highcharts configuration object for the managed chart. May include
+     * @param {Object} c.highchartsConfig - Highcharts configuration object for the managed chart. May include
```

Comment:
💥 I think we should document in changelog as a breaking change since we're taking breaking changes with this release anyway.

Given that we have relatively few charts in production usage, I would also favor throwing in the ctor vs. swapping the configs to keep things working.  (Typically I lobby for the smoother deprecations when we know we're gonna hit a lot of them, but I think we can be more aggressive here.)